dnf-behave-tests/dnf/comps-group-disabled-repo.feature

```diff
@@ -0,0 +1,43 @@
+@bz2064341
+Feature: Test group remove when repositories are disabled
+
+Background: Install group DNF-CI-Testgroup
+    Given I use repository "dnf-ci-thirdparty"
+      And I use repository "dnf-ci-thirdparty-2"
+      And I use repository "dnf-ci-fedora"
+     When I execute dnf with args "group install DNF-CI-Testgroup"
+     Then the exit code is 0
+      And Transaction is following
+          | Action        | Package                           |
+          | install-group | filesystem-0:3.9-2.fc29.x86_64    |
+          | install-group | lame-0:3.100-4.fc29.x86_64        |
+          | install-dep   | setup-0:2.12.1-1.fc29.noarch      |
+          | install-dep   | lame-libs-0:3.100-4.fc29.x86_64   |
+          | group-install | DNF-CI-Testgroup                  |
+     When I execute dnf with args "group install DNF-CI-Testgroup-2"
+     Then the exit code is 0
+      And Transaction is following
+          | Action        | Package                           |
+          | group-install | DNF-CI-Testgroup-2                |
+
+Scenario: Remove group with disabled repository
+   When I execute dnf with args "group remove DNF-CI-Testgroup --disablerepo=\*dnf-ci-thirdparty"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                           |
+        | remove        | filesystem-0:3.9-2.fc29.x86_64    |
+        | remove        | lame-0:3.100-4.fc29.x86_64        |
+        | remove-unused | setup-0:2.12.1-1.fc29.noarch      |
+        | remove-unused | lame-libs-0:3.100-4.fc29.x86_64   |
```

Comment:
This is weird. May be I'm missing something, but:
1. `filesystem` and `lame` packages are part of both `DNF-CI-Testgroup` and `DNF-CI-Testgroup-2`
2. both groups are installed
3. when you try to remove one of the groups, the `filesystem` and `lame` packages should stay put on the system - because they are part of the other group

According to man dnf (group remove: Mark the group removed and remove those packages in the group  from  the system  which  do not belong to another installed group and were not installed explicitly by the user.) only the group should be removed, not the packages.

I didn't find any scenario testing this case (dnf group remove does not remove packages that belong to another installed group) in our suite.